diff --git a/cmake/SDL3Config.cmake.in b/cmake/SDL3Config.cmake.in
--- a/cmake/SDL3Config.cmake.in
+++ b/cmake/SDL3Config.cmake.in
@@ -14,8 +14,5 @@
 # Find SDL3::Headers
 if(NOT TARGET SDL3::Headers)
   include("${CMAKE_CURRENT_LIST_DIR}/SDL3headersTargets.cmake")
-  if(NOT CMAKE_VERSION VERSION_LESS "3.25")
-    set_property(TARGET SDL3::Headers PROPERTY SYSTEM 0)
-  endif()
 endif()
 set(SDL3_Headers_FOUND TRUE)
